lib/common_test/doc/src/ct.xml

```diff
@@ -43,14 +43,6 @@
        tests and basic functions for <c>Common Test</c> case issues, such as
        configuration and logging.</p>
 
```

Comment:
Please add a warning mentioning config macro as deprecated and not recommended to be used in new test suites.
For now, we would like to have some trace in docs, for readers who might still see it in their source code.

Reply:
I added a warning in 710df91cb5f4c9cfc037c87fad0ca91198130930. As usual, let me know if I should squash the commits